holmes/core/tools.py

```diff
@@ -131,6 +137,20 @@ class Tool(ABC, BaseModel):
         None  # templated string to show to the user describing this tool invocation (not seen by llm)
     )
     additional_instructions: Optional[str] = None
+    transformer_configs: Optional[List[Dict[str, Any]]] = None
```

Comment:
Consider: use pydantic models instead of implementing it with python types. That way you will now need to implement all validaitons manually in validation.py

Reply:
done.